apollos-church-api/src/data/ActionAlgorithm.js

```diff
@@ -32,6 +33,22 @@ class dataSource extends ActionAlgorithm.dataSource {
     }));
   }
 
+  async openURLContentFeedAlgorithm(...args) {
```

Comment:
Should we name this algorithm something more specific since `'GoDoGood://app-link/nav/Go'` is being hard-coded?

Reply:
Either that, or I think this URL should be loaded from the content-item in rock and not hard-coded